src/fieri/app/models/foodcritic_worker.rb

```diff
@@ -23,12 +23,26 @@ def make_post(params, feedback, status)
       fieri_key: ENV['FIERI_KEY'],
       cookbook_name: params['cookbook_name'],
       cookbook_version: params['cookbook_version'],
-      foodcritic_feedback: feedback,
+      foodcritic_feedback: format_feedback(feedback, status),
       foodcritic_failure: status
     )
     unless response.is_a? Net::HTTPSuccess
       error_msg = "Unable to POST Foodcritic Evaluation of #{params[cookbook_name]} " + response.message
       raise error_msg
     end
   end
+
+  private
+
+  def foodcritic_info
+    "Run with Foodcritic Version #{FoodCritic::VERSION} with tags #{ENV['FIERI_FOODCRITIC_TAGS']}"
```

Comment:
At least according to my understanding of the foodcritic help and [how we call foodcritic in fieri](https://github.com/chef/supermarket/blob/master/src/fieri/app/models/cookbook_artifact.rb#L43-L45) with both `-f` and `-t`:
```
foodcritic [cookbook_paths]
    -t, --tags TAGS                  Check against (or exclude ~) rules with the specified tags.
    -f, --epic-fail TAGS             Fail the build based on tags. Use 'any' to fail on all warnings.
```